presto-docs/src/main/sphinx/connector/hana.rst

```diff
@@ -24,6 +24,40 @@ connection properties as appropriate for your setup:
     connection-user=root
     connection-password=secret
 
+Connection security
+------------------
+
+The JDBC driver and connector automatically use Transport Layer Security (TLS) encryption and certificate validation. This requires a suitable TLS certificate configured on your Hana database host.
+
+To disable encryption in the connection string, use the ``encrypt`` property:
+
+.. code-block:: none
+
+    connection-url=jdbc:sap://<host>:<port>?encrypt=false;
+
+Other SSL configuration properties that can be configured using the ``connection-url``:
+
+SSL Configuration Properties
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+================================================== ==================================================================== ===========
+Property Name                                      Description                                                          Default
+================================================== ==================================================================== ===========
+``validateCertificate``                            Indicates that the SSL certificate presented by the server            ``true``
+                                                   should be validated against the truststore specified.
+
+``trustStoreType``                                 File format of the truststore file, for example ``jks``.
```

Comment:
```suggestion
``trustStoreType``                                 File format of the truststore file, for example ``JKS`` or ``PEM``.
```
Suggest consistency with your recent work in [PR 21938](https://github.com/prestodb/presto/pull/21938).

Reply:
@steveburnett I don't think PEM is supported for saphana.

Reply:
Got it, thanks! Appreciate the clarification.